fv/containers/containers.go

```diff
@@ -201,3 +235,24 @@ func (c *Container) CanConnectTo(ip, port string) bool {
 
 	return err == nil
 }
+
+func RunEtcd() *Container {
+	return Run("etcd",
+		"quay.io/coreos/etcd",
+		"etcd",
+		"--advertise-client-urls", "http://127.0.0.1:2379",
```

Comment:
Hmm, this should be the real IP of the etcd container or a smart client may end up trying to reconnect to 127.0.0.1 instead of the main etcd IP.  Is that easy to fix?

Reply:
Unfortunately I don't see any nice way of doing this.  We could start using `--ip=` and managing all the Docker container IPs ourselves, but obviously that would be far from nice.  All of the examples that I could find about running etcd in a container use `--net=host`, so they already know the host's IP.